plugins/site-example/build.gradle

```diff
@@ -1,27 +0,0 @@
-/*
```

Comment:
@rjernst there is a link in the documentation to this file: https://www.elastic.co/guide/en/elasticsearch/plugins/master/plugin-authors.html
Is that something you could fix?